tests/floats.rs

```diff
@@ -1,8 +1,30 @@
-use ron::de::from_str;
+use ron::{
+    de::from_str,
+    ser::{to_string_pretty, PrettyConfig},
+};
 
 #[test]
 fn test_inf_and_nan() {
     assert_eq!(from_str("inf"), Ok(std::f64::INFINITY));
     assert_eq!(from_str("-inf"), Ok(std::f64::NEG_INFINITY));
     assert_eq!(from_str::<f64>("NaN").map(|n| n.is_nan()), Ok(true))
 }
+
+#[test]
+fn decimal_floats() {
+    let pretty = PrettyConfig::new().with_decimal_floats(false);
+    let without_decimal = to_string_pretty(&1.0, pretty).expect("Serialization failed");
```

Comment:
Do we need those `expect` strings here? For tests, I think `unwrap` is perfectly fine.

Reply:
I was following the pattern in the "preserve_sequence" tests, but I'm happy to swap these out. "preserve_sequence" appears to be the only existing test that uses expect.